src/kaleidoscope/key_defs.h

```diff
@@ -36,7 +36,7 @@ class Key {
 
   Key() = default;
 
-  constexpr Key(uint16_t raw)
+  constexpr Key(uint16_t raw) // NOLINT(runtime/explicit)
```

Comment:
We very much do not want to force explicit conversion here, because that would make some of the callsites hell to decipher.